glue/main.py

```diff
@@ -115,3 +114,10 @@ def load_plugins(splash=None, require_qt_plugins=False, plugins_to_load=None):
     # that were previously read.
     from glue._settings_helpers import load_settings
     load_settings()
+
+
+def list_plugins():
```

Comment:
Thinking about this, I think we might want to actually have ``list_available_plugins`` and ``list_loaded_plugins`` as there are use cases for both. Could you implement both here?